tests/integration_tests/test_botorch.py

```diff
@@ -84,6 +84,28 @@ def candidates_func(
     assert candidates_func_call_count == n_trials - n_startup_trials
 
 
+def test_botorch_qnehvi_candidates_func() -> None:
+    # This test is needed because qnehvi_candidates_func is never chosed as a default.
+    # The other funcs can be chosed as a default and hence
+    # covered by test_botorch_candidates_func_none.
```

Comment:
```suggestion
    # This test is needed because qnehvi_candidates_func is never chosen as a default.
    # The other funcs can be chosen as a default and hence
    # covered by test_botorch_candidates_func_none.
```
Nit: typo in comments.